distributions/openhab/src/main/filtered-resources/userdata/etc/version.properties

```diff
@@ -7,6 +7,7 @@ Repository        Version
 ----------------------------------------
 openhab-distro  : ${project.version}
 openhab-core    : ${ohc.version}
+openhab-webui   : ${ohw.version}
```

Comment:
remove this one (UIs are still considered a part of openhab2-addons, just as zigbee and zwave bindings)